site/docs/master/install-overview.md

```diff
@@ -23,6 +23,8 @@ velero install \
     [--pod-annotations] \
 ```
 
+The `--secret-file` argument is optional if using node-based IAM policies.
```

Comment:
Probably worth noting here as well that `--no-secret` needs to be specified if not using `--secret-file`.